.github/workflows/scripts/ping-codeowners-prs.sh

```diff
@@ -57,7 +57,7 @@ main() {
         -H "Authorization: Bearer ${GITHUB_TOKEN}" \
         "https://api.github.com/repos/${REPO}/pulls/${PR}/requested_reviewers" \
         -d "{\"reviewers\":[${REVIEWERS}]}" \
-        | jq ".message" \
+        || jq ".message" \
```

Comment:
I don't think this is right, we want to pipe the output of `curl` to `jq` to get the value on the `message` field and print it